docs/usage/export.md

```diff
@@ -1,5 +1,57 @@
-Exporting data
+Exporting Data
 ==============
```

Comment:
All the headers in this file seem to be capitalised like titles. I think this title is the only place that should be capitalised this way.